src/main/java/step3/runType/RunStrategy.java

```diff
@@ -0,0 +1,21 @@
+package step3.runType;
+
+public interface RunStrategy {
+
+    int NO_PROGRESS_DISTANCE = 0;
+    int NORMAL_PROGRESS_DISTANCE = 1;
+    int HIGH_SPEED_PROGRESS_DISTANCE = 2;
+
+    int ENGINE_POWER_LIMIT = 9;
+    int HIGH_POWER_BOUNDARY = 4;
+
+    int run();
+
+    default boolean canProgress() {
+        int enginePower = (int) (Math.random() * ENGINE_POWER_LIMIT);
+        if (enginePower >= HIGH_POWER_BOUNDARY) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return enginePower >= HIGH_POWER_BOUNDARY;
```
이럴땐 이렇게 바로 사용할 수 있답니다.

Reply:
감사합니다!  수정하면서 코드가 바뀌어 쓸일은 없어졌지만 비슷한 상황이 오면 그때 적용해 보겠습니다.